tools/data-handler/src/move.ts

```diff
@@ -61,6 +70,10 @@ export class Move {
       throw new Error(`Card ${destination} not found from project`);
     }
 
+    if (destinationCard.path.includes(source)) {
```

Comment:
not in the scope of this, but: I think our path includes the full path on the system. Should we actually instead use path relative to project root? Having the whole system path might produce some weird bugs, if user has some folders, which have a meaning to us, higher up in the path

Reply:
`Card` and `Resource` paths are indeed full paths in system. 
Both paths could be relative to the projectRoot.

Not going to change it here, but I'll make a JIRA ticket about it.